.github/actions/create_manifest/action.yml

```diff
@@ -0,0 +1,43 @@
+name: 'Create manifest'
+description: 'Creates manifest containing versions of the product and the corresponding repositories'
+inputs:
+  repos:
+    description: "Multi-line list of repositories to include to manifest"
+    required: true
+  product_type:
+    description: "Unique string to reflect product configuration"
+    required: true
+  save_to:
+    description: "Path to save manifest to"
+    required: true
+  action_path:
+    description: "Action path, if not set - taken from github context"
+    required: false
+  target_arch:
+    description: "Target architecture"
+    required: true
+  build_type:
+    description: "Build type: release | debug | release_with_debug"
+    required: true
+
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install Python dependencies
+      uses: py-actions/py-dependency-install@v4
```

Comment:
What is the benefit to use 3rdparty action instead of pip install -r?

Reply:
Handles pip upgrades as well (you can see how it works in smart ci jobs)

Reply:
setup python action should update the pip too

Reply:
let's try pip install instead or we would need to set action commit hash